hls-plugin-api/src/Ide/Types.hs

```diff
@@ -113,6 +152,8 @@ instance Show (IdeCommand st) where show _ = "<ide command>"
 data PluginDescriptor (ideState :: *) =
   PluginDescriptor { pluginId           :: !PluginId
                    -- ^ Unique identifier of the plugin.
+                   , pluginPriority     :: Natural
```

Comment:
What happens if two plugins have the same priority?

Reply:
Unspecified order as decided by unordered-containers